### ISSUE TITLE
Consider removing `touch-action: manipulation` rules in favor of a responsive meta tag

### ISSUE THREAD

**Issue by kripod** [csstools/sanitize.css#201]
As seen [on web.dev](https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away) and observed [in a real project](https://github.com/logux/logux.io/pull/8), adding the following meta tag should be sufficient to bypass 300ms tap delays:

```html
<meta name="viewport" content="width=device-width">
```

I would suggest removing the following code [from sanitize.css](https://github.com/csstools/sanitize.css/blob/66dd0308db731b956f0f78501d1f1386fb952a25/sanitize.css#L498-L515), leaving a remark about the HTML-based technique in the readme:

```css
a,
area,
button,
input,
label,
select,
summary,
textarea,
[tabindex] {
  -ms-touch-action: manipulation; /* 1 */
  touch-action: manipulation; /* 2 */
}
```

**Comment by tigt** [comment]
I’d love this, but I am somewhat concerned just that I don’t know the full extent of what older browsers the `meta[name=viewport]` technique doesn’t work on.

**Comment by jonathantneal** [comment]
Thankfully, we have this data:

https://patrickhlauke.github.io/touch/tests/results/

The `touch-action` will be removed in the next version, while `-ms-touch-action: manipulation` is left for IE10.

There will also be an evergreen version of the file, which will remove all IE styles.

**Comment by kripod** [comment]
Great, thank you for the feedback!

**Comment by jonathantneal** [comment]
Fixed in e444e65
